cpu/kinetis_common/include/periph_cpu.h

```diff
@@ -309,10 +309,18 @@ enum {
 /** @} */
 
 /**
+ * @brief UART hardware module types
+ */
+enum {
+    KINETIS_UART,   /**< Kinetis UART module type */
+    KINETIS_LPUART, /**< Kinetis Low-power UART (LPUART) module type */
+};
+
+/**
  * @brief UART module configuration options
  */
 typedef struct {
-    UART_Type *dev;               /**< Pointer to module hardware registers */
+    void *dev;                    /**< Pointer to module hardware registers */
```

Comment:
Yes, this is part of the commit to support both UART and LPUART in the same driver, and there are other CPUs which only have one or the other, so I used a void pointer as the least common denominator.